src/main/java/com/redhat/red/build/finder/DistributionAnalyzer.java

```diff
@@ -15,12 +15,6 @@
  */
 package com.redhat.red.build.finder;
 
-import java.io.ByteArrayInputStream;
```

Comment:
Our IDEs seem to be fighting about import order. This appears in multiple files now.

Reply:
Yuck. IntelliJ has variety of preferences for it ; I guess you're using Eclipse? I'm using the JBoss IntelliJ formatter.

Reply:
This is trivial to fix but need to agree on a standardised formatter ;-)

Reply:
Yes, Eclipse, and IntelliJ wasn't open source at the time.

Eclipse has this order: java, javax, org, com. IntelliJ seems to put java and javax last.

Reply:
It seems a bit more complicated than this as there are also differences in blank lines between imports. See: http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.html

Reply:
It seems that IntelliJ is more configurable. In terms of inserting blank lines, I can't find an option in Eclipse. It appears that a blank line in always inserted.

Reply:
The default for checkstyle seems closer to the eclipse defaults. See: http://checkstyle.sourceforge.net/config_imports.html#ImportOrder